### PR TITLE
LPS-128049 The bottom part of the scroll bar is not displayed in the Rich Text field

### PR DESCRIPTION
**Pull request by boton**
**Test plan:** 

At this moment you need to revert this commit `60f8e5d713198018d9cece0f17b474a1f07526b4` and deploy `data-engine/data-engine-taglib`.
**Steps to reproduce:** https://issues.liferay.com/browse/LPS-128049

**After fix:**
image

### CODE REVIEW

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp [hunk 1]
```diff
@@ -369,9 +369,7 @@ renderResponse.setTitle(headerTitle);
 										%>
 
 											<c:if test="<%= (curDLFileEntryType.getFileEntryTypeId() == DLFileEntryTypeConstants.FILE_ENTRY_TYPE_ID_BASIC_DOCUMENT) || (fileEntryTypeId == curDLFileEntryType.getFileEntryTypeId()) || DLFileEntryTypePermission.contains(permissionChecker, curDLFileEntryType, ActionKeys.VIEW) %>">
-
-											<aui:option label="<%= HtmlUtil.escape(curDLFileEntryType.getName(locale)) %>" selected="<%= fileEntryTypeId == curDLFileEntryType.getPrimaryKey() %>" value="<%= curDLFileEntryType.getPrimaryKey() %>" />
-
```
Comment (4lejandrito) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:374]: Why do we need to add these spaces? I searched the codebase and we normally don't insert empty lines after `c:if`s.

**Comment by boton** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp [hunk 2]
```diff
@@ -422,40 +420,35 @@ renderResponse.setTitle(headerTitle);
...
 
-										ddmFormValues = dlEditFileEntryDisplayContext.getDDMFormValues(fileEntryMetadata.getDDMStorageId());
-									}
-									catch (Exception e) {
-									}
-							%>
+										try {
+											DLFileEntryMetadata fileEntryMetadata = DLFileEntryMetadataLocalServiceUtil.getFileEntryMetadata(ddmStructure.getStructureId(), fileVersionId);
 
-										<c:if test="<%= !dlEditFileEntryDisplayContext.isDDMStructureVisible(ddmStructure) %>">
```
Comment (4lejandrito) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:441]: Why are we removing this `c:if`s? I can't see why removing them is needed to fix your issue. If it is needed, can you do it in a separate commit explaining the reasoning behind the change?

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp (hunk 2 above)
Comment (4lejandrito) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:441]: Oh I see what you've one. You're conditionally adding the class. I got confused. I still think separating this in 2 different commits would have helped me to understand why you're doing this.

Comment (boton) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:441]: Updated the commits

**Review by 4lejandrito — APPROVED** [review]

**Comment by boton** [comment]
I don't know the cause of weird formating. The `formatSource` command complaint about indentation and entry lines.

```
> Task :apps:document-library:document-library-web:formatSource
modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp (SourceCheck:JSPEmptyLinesCheck)
modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp (SourceCheck:JSPIndentationCheck)
```

I isolate the SF in 6149c04a3fc31ceb788b4ae3215667cf8850f27f

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**